README.md

```diff
@@ -170,6 +170,20 @@ Set the var `clojure-ts-indent-style` to change it.
 >
 > You can find [this article](https://metaredux.com/posts/2020/12/06/semantic-clojure-formatting.html) comparing semantic and fixed indentation useful.
 
+#### Customizing semantic indentation
+
+You can customize how body of different forms is indented. There is a set of
+default rules, which are aligned with cljfmt indent rules, but you can also set
+custom rules as:
+
+```emacs-lisp
+(setopt clojure-ts-semantic-indent-rules '(("are" . (:block 1))))
```

Comment:
Might be better to have several forms here, so it's clearer what the format is.

Reply:
I think it'd be good to explain also the various `block` and `inner` values here, similar to how it's done in `clojure-mode`'s docs.

Reply:
I've adapted some examples from `clojure-mode` documentation and added some explanation about the rules.